var/spack/repos/builtin/packages/llvm/package.py

```diff
@@ -683,16 +683,16 @@ def cmake_args(self):
                 # hence the test to see if the version starts with "flang".
                 targets.append('CppBackend')
 
-            if 'x86' in spec.architecture.target.lower():
+            if 'x86' in spec.target.family:
```

Comment:
```suggestion
            if spec.target.family == 'x86' or spec.target.family == 'x86_64'
```
This is to be super-defensive. In `llvm` the `X86` target means both 64-bit and 32-bit machines. I don't think anybody is using Spack to build software on a `pentium` but who knows :smile: